transaction/core/src/ring_signature/generator_cache.rs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2022 The MobileCoin Foundation
```

Comment:
```suggestion
// Copyright (c) 2018-2022 The MobileCoin Foundation
```

Reply:
see i feel like this is wierd, this file was written in 2022 so how can we claim it is copyrighted since 2018?

in another open source project i worked on the files all had different starting years and the same ending year for copyright claims. i'm happy to change it, i just want to make sure I understand why we are doing this